### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by IgorOlifirenko**

### CODE REVIEW

**Review by Oleksandr563487953 — APPROVED** [review]

File: tests/test_main.py [hunk 1]
```diff
@@ -1 +1,31 @@
...
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_predicted_rate_within_reference(
+        mocked_get_exchange_rate_prediction
+):
+    mocked_get_exchange_rate_prediction.return_value = 81
+
+    assert cryptocurrency_action(85) == "Do nothing"
+    assert cryptocurrency_action(78) == "Do nothing"
```
Comment (MasterpieceElbow) [inline tests/test_main.py:31]: Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 or 0.95 * value passed in `cryptocurrency_action`.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]
